setup.cfg

```diff
@@ -53,6 +53,8 @@ install_requires =
     statsmodels==0.11.1
     matplotlib==3.2.1
     cvxpy==1.1.1
+    cython==0.29.21
```

Comment:
I am not so sure about adding these 2 as hard dependencies. 

Ok, let's do it, we are going to solve this problem with the Docker containers.

Make double sure the unit tests that use POT are very robust! If there is a dependency problem on their side, I want our tests to pick up that the code broke.

Reply:
Ok, we can later transfer them to a soft dependency if needed (or if these methods won't be popular).